lib/std/zig/system/NativeTargetInfo.zig

```diff
@@ -578,7 +578,7 @@ fn glibcVerFromSoFile(file: fs.File) !std.SemanticVersion {
     return max_ver;
 }
 
-fn glibcVerFromLinkName(link_name: []const u8, prefix: []const u8) !std.SemanticVersion {
+fn glibcVerFromLinkName(link_name: []const u8, prefix: []const u8) error{ UnrecognizedGnuLibCFileName, InvalidGnuLibCVersion }!std.SemanticVersion {
```

Comment:
Not really your change, but might be worth adding a couple tests for glibcVerFromLinkName.

Reply:
Done (also added small doctests to std.zig.CrossTarget.parseVersion).